### PR TITLE
Fix: terminate process on entering multiple slash

### LINKED ISSUE #683: Entering multiple '/' triggers HTTP requests to the back end.
**Describe the bug**
Entering multiple '/' triggers HTTP requests to the back end. 

**To Reproduce**
1. Open dev tools
2. Enter https://graph.microsoft.com/v1.0/me//
3. Notice two network calls are made

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
 
[AB#7485](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/7485)

### PR DESCRIPTION
**Pull request by ezkemboi**
- fixes #683

## Overview

Brief description of what this PR does, and why it is needed.

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as bundling scripts, restarting services, etc.
* Include test case, and expected output

### CODE REVIEW

**Comment by ezkemboi** [comment]
Cc. @thewahome

**Comment by ddyett** [comment]
@thewahome  can you describe the bad from the original issue.  I'm less concerned that it fired and failed as expected but it seems that if you backspace we don't get suggestions again.  Fixing that would be good.

Also note that this call in graph explorer does work
https://graph.microsoft.com/v1.0/me//?$select=displayname

**Comment by ezkemboi** [comment]
@ddyett , I am going ahead to work on fixing that backspacing issue.

**Comment by ezkemboi** [comment]
Removed double slash fix and added a fix for autosuggestion on backspacing. 

Cc. @thewahome and @ddyett

**Comment by thewahome** [comment]
@ezkemboi the force push seems to have overwritten the previous slash changes...

**Comment by ezkemboi** [comment]
@thewahome, I have added it back.

File: src/app/views/query-runner/query-input/auto-complete/AutoComplete.tsx [hunk 1]
```diff
@@ -182,20 +186,12 @@ class AutoComplete extends Component<IAutoCompleteProps, IAutoCompleteState> {
...
     }
 
-    let filteredSuggestions = parametersWithVerb.links;
-    if (compare) {
-      filteredSuggestions = filteredSuggestions.filter((suggestion: string) => {
-        return suggestion.toLowerCase().indexOf(compare.toLowerCase()) > -1;
-      });
-    }
-
-    this.setSuggestions(filteredSuggestions);
```
Comment (thewahome) [inline src/app/views/query-runner/query-input/auto-complete/AutoComplete.tsx:198]: @ezkemboi why was this removed?

Comment (thewahome) [reply src/app/views/query-runner/query-input/auto-complete/AutoComplete.tsx:197]: Found that the branch was just behind the main repo's dev.

**Review by thewahome — APPROVED** [review]

**Pull request merged**